lib/svgo/coa.js

```diff
@@ -295,9 +304,11 @@ function optimizeFromString(svgstr, config, datauri, input, output) {
                 output = input;
             }
 
-            console.log('\r');
+            if (! config.quiet) {
```

Comment:
Oops, I apologize. I have made sure that the source code does not contain /!\s+/ anymore.